geomesa-quickstart-python/geomesa_quickstart_jnius.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+#coding:utf-8
+"""
+geomesa_quickstart_jnius.py
+
+Note: this has been tested with python 2.7.12 & 3.5
+
+Description:
+   This script is a python adaptation of the GeoMesa accumulo quickstart. It is a querry only
+   demo, in contrast to the java quickstart that creates the table & loads data into it. This
+   script uses two second party packages (jnius & bokeh) that need to be loaded to run. 'jnius'
```

Comment:
Like how we style geomesa -> GeoMesa. I think bokeh -> Bokeh and jnius -> JNIus but from looking did they changed the name of JNIus to [Pyjnius](https://github.com/kivy/pyjnius)?